calcom/cal.com
# feat: Shows link location and respective icon in /bookings

## What does this PR do?

Shows link location and respective icon in /bookings.
Fixes #11730 
/claim #11730 

![Screenshot from 2023-12-13 07-25-10](https://github.com/calcom/cal.com/assets/41051387/d40acff7-4470-4e59-bc70-1da8039725c5)


## Type of change




## How should this be tested?

Once a booking is created, go to the /bookings page and observe the link shown in booking card.

## Mandatory Tasks
